src/main/java/se/sundsvall/businessengagements/config/CacheConfig.java

```diff
@@ -0,0 +1,49 @@
+package se.sundsvall.businessengagements.config;
+
+import static se.sundsvall.businessengagements.integration.party.PartyIntegration.LEGAL_ID_CACHE;
+import static se.sundsvall.businessengagements.integration.party.PartyIntegration.PARTY_ID_CACHE;
+import static se.sundsvall.businessengagements.service.BusinessEngagementsService.BUSINESS_ENGAGEMENTS_CACHE;
+import static se.sundsvall.businessengagements.service.BusinessEngagementsService.BUSINESS_INFORMATION_CACHE;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.cache.support.NoOpCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@EnableCaching
+@Configuration
+public class CacheConfig {
```

Comment:
Do you really need this class at all?

Reply:
Somehow I messed up the config and it didn't work at first so I went this route.
Tried with only config and now it works. I'll remove these classes.